lib/extensions/src/lib/components/dynamic-component/dynamic.component.spec.ts

```diff
@@ -90,6 +91,13 @@ describe('DynamicExtensionComponent', () => {
             const testComponent = fixture.debugElement.query(By.css('test-component')).componentInstance;
             expect(testComponent.data).toBe(data);
         });
+
+        it('should assign menuItem from dynamically generated component in ngAfterViewInit', () => {
+            const innerElement = fixture.debugElement.query(By.css('[data-automation-id="found-me"]'));
```

Comment:
I see the same selector in 76th line, can we move it into some function and reuse?

Reply:
done